packages/patternfly-4/react-integration/demo-app-ts/src/components/demos/DrawerDemo/DrawerDemo.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
```

Comment:
Can you also update the page demo so we exercise the the new interfaces?

Reply:
Added the interfaces to drawer demo.